test/integration/util/util.go

```diff
@@ -81,6 +81,18 @@ func (m *MinishiftRunner) Start() {
 	m.RunCommand(fmt.Sprintf("start %s", m.CommandArgs))
 }
 
+func (m *MinishiftRunner) NeedsSetup() {
```

Comment:
@hferentschik  or `NeedCdkSetup` ? it just rhyme with isCDK then `NeedCdkSetup` :) Just my thought (not a hard feeling around it).